src/main/java/com/epam/prd/jjop/helpers/arrays/Print.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.prd.jjop.helpers.arrays;
+
+import java.util.Arrays;
+
+/**
+ * Examples of printing arrays with different number of dimensions using {@link Arrays#toString(Object[])}
+ * and {@link Arrays#deepToString(Object[])} methods.
+ *
+ * @author Anna Silenko
+ * @see java.util.Arrays
+ */
+public class Print {
+
+    public static void main(String[] args) {
+
+        String[] array1D = {"=^._.^= ∫", "(=^･ｪ･^=))ﾉ彡☆", "(⌯’▾’⌯) ∫"};
+
+        String[][] array2D = {{"001100"},
+                {"010010"},
+                {"011110"},
+                {"100001"},
+                {"101101"},
+                {"110011"},};
+
+        int[][][] array3D = {{{0}, {0}, {0}},
+                {{1}, {1}, {1}},
+                {{2}, {2}, {2}}};
+
+        /**
+         * <p>If we try to call System.out.println() method on an array, {@link Object#toString()} method will be called,
+         * which returns:</p>
+         * <code>getClass().getName() + "@" + Integer.toHexString(hashCode())</code>
+         * <p>So for a Strings array outputs will be something like:
```

Comment:
```suggestion
         * <p>So for a strings array outputs will be something like:
```

there is no class String in Java :D